.github/workflows/ci.yml

```diff
@@ -23,29 +23,31 @@ jobs:
         export MAKEFLAGS=-j3
         conda create --name build
         source activate build
-        conda-build -c conda-forge plumed
-        conda-build -c conda-forge gromacs
-        conda-build -c conda-forge lammps
+        conda-build -m .github/conda_build_config.yaml -c conda-forge plumed
```

Comment:
Useful trick, so that we have them in a single place, thanks!

Reply:
Yes! And this is intentionally not in the repository root, to prevent it from being picked on local builds: https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#conda-build-variant-config-files